tests/generator/BGenTests.cs

```diff
@@ -260,28 +260,28 @@ public void Bug35176 ()
 			string expectedAttributes =
 @"	Bug35176.IFooInterface: [SupportedOSPlatform(""ios14.3"")]
 	Bug35176.IFooInterface: [SupportedOSPlatform(""maccatalyst15.3"")]
-	Bug35176.IFooInterface: [SupportedOSPlatform(""macos12.2"")]
+	Bug35176.IFooInterface: [SupportedOSPlatform(""macos16.2"")]
```

Comment:
@rolfbjarne macos16? looks like you came from a parallel universe where Apple didn't go to macOS 26 :) 

did you mean macos16.2 -> macos14.0?